src/controller/cve.controller/index.js

```diff
@@ -81,25 +68,22 @@ router.get('/cve',
   /*
   #swagger.tags = ['CVE Record']
   #swagger.operationId = 'cveGetFiltered'
-  #swagger.summary = "Retrieves all CVE records (accessible to all users)"
+  #swagger.summary = "Retrieves all CVE entries (accessible to all users)"
   #swagger.description = "
         <h2>Access Control</h2>
         <p>Endpoint is accessible to all</p>
         <h2>Expected Behavior</h2>
-        <p><b>All users:</b> Retrieves all CVE records for all organizations</p>"
+        <p><b>All users:</b> Retrieves all CVE entries for all organizations</p>"
   #swagger.parameters['$ref'] = [
     '#/components/parameters/cveRecordFilteredTimeModifiedLt',
     '#/components/parameters/cveRecordFilteredTimeModifiedGt',
     '#/components/parameters/cveState',
     '#/components/parameters/countOnly',
     '#/components/parameters/assignerShortName',
     '#/components/parameters/assigner',
-    '#/components/parameters/apiEntityHeader',
-    '#/components/parameters/apiUserHeader',
-    '#/components/parameters/apiSecretHeader'
   ]
   #swagger.responses[200] = {
-    description: 'A list of CVE records and pagination fields, only allowed by the Secretariat.',
+    description: 'A list of CVE records and pagination fields, only allowed by the Secretariat',
     content: {
```

Comment:
cve_records should look like this
        {
            "dataType": "CVE_RECORD",
            "dataVersion": "5.0",
            "cveMetadata": {
                "cveId": "CVE-2022-0182",
                "assignerOrgId": "ee1fa4c6-8d34-4353-ad97-194c1b986b8b",
                "state": "PUBLISHED",
                "assignerShortName": "exampleCNA",
                "requesterUserId": "adfb94fe-a90a-42a5-8ce4-12aa1f897cde",
                "dateReserved": "2022-04-04T02:28:12.844Z",
                "datePublished": "2022-04-04T02:29:23.283Z"
            },
            "containers": {
                "cna": {
                    "x_a": {
                        "bb": "c"
                    },
                    "affected": [
                        {
                            "product": "p",
                            "vendor": "v",
                            "versions": [
                                {
                                    "version": "1.2",
                                    "status": "affected"
                                }
                            ]
                        }
                    ],
                    "descriptions": [
                        {
                            "lang": "en",
                            "value": "v p 1.2 is insecure."
                        }
                    ],
                    "problemTypes": [
                        {
                            "descriptions": [
                                {
                                    "description": "insecurity",
                                    "lang": "en"
                                }
                            ]
                        }
                    ],
                    "providerMetadata": {
                        "orgId": "ee1fa4c6-8d34-4353-ad97-194c1b986b8b",
                        "shortName": "exampleCNA"
                    },
                    "references": [
                        {
                            "url": "https://example.com"
                        }
                    ]
                }
            }
        },